DEHPEcosimPro/ViewModel/DstBrowserHeaderViewModel.cs

```diff
@@ -24,13 +24,199 @@
+    using ReactiveUI;
 
     /// <summary>
-    /// The <see cref="DstBrowserHeaderViewModel"/> is the view model the <see cref="DstBrowserHeader"/>
+    /// The view model for <see cref="DstBrowserHeader"/>
     /// </summary>
-    public class DstBrowserHeaderViewModel : IDstBrowserHeaderViewModel
+    public class DstBrowserHeaderViewModel : ReactiveObject, IDstBrowserHeaderViewModel
     {
+        /// <summary>
+        /// The <see cref="IDstController"/>
+        /// </summary>
+        private readonly IDstController dstController;
+
+        /// <summary>
+        /// The <see cref="IStatusBarControlViewModel"/> instance
+        /// </summary>
+        private readonly IStatusBarControlViewModel statusBarControlViewModel;
+
+        /// <summary>
+        /// Backing field for <see cref="ServerAddress"/>
+        /// </summary>
+        private string serverAddress;
+
+        /// <summary>
+        /// The <see cref="NodeId"/> of the ServerStatus.CurrentTime node in an OPC session
+        /// </summary>
+        private readonly NodeId currentServerTimeNodeId = new NodeId(Variables.Server_ServerStatus_CurrentTime);
+
+        /// <summary>
+        /// Gets or sets the URI of the connected data source
+        /// </summary>
+        public string ServerAddress
+        {
+            get => this.serverAddress;
+            set => this.RaiseAndSetIfChanged(ref this.serverAddress, value);
+        }
+
+        /// <summary>
+        /// Backing field for <see cref="SamplingInterval"/>
+        /// </summary>
+        private int samplingInterval;
+
+        /// <summary>
+        /// Gets or sets the time, in milliseconds, between which data is recorded
+        /// </summary>
+        public int SamplingInterval
+        {
+            get => this.samplingInterval;
+            set => this.RaiseAndSetIfChanged(ref this.samplingInterval, value);
+        }
+
+        /// <summary>
+        /// Backing field for <see cref="VariablesCount"/>
+        /// </summary>
+        private int variablesCount;
+
+        /// <summary>
+        /// Gets or sets the total number of variables in the open session
+        /// </summary>
+        public int VariablesCount
+        {
+            get => this.variablesCount;
+            set => this.RaiseAndSetIfChanged(ref this.variablesCount, value);
+        }
+
+        /// <summary>
+        /// Backing field for <see cref="ServerStartTime"/> 
+        /// </summary>
+        private DateTime? serverStartTime;
+
+        /// <summary>
+        /// Gets or sets the date and time, in UTC, from which the server has been up and running
+        /// </summary>
+        public DateTime? ServerStartTime
+        {
+            get => this.serverStartTime;
+            set => this.RaiseAndSetIfChanged(ref this.serverStartTime, value);
+        }
+
+        /// <summary>
+        /// Backing field for <see cref="CurrentServerTime"/> 
+        /// </summary>
+        private DateTime? currentServerTime;
```

Comment:
Please group private fields together and public properties together. The private fields should be at the top of file, while Properties come directly after constructors. You can setup Resharper to do this for you and be taken in to account as part of the Cleanup function.